sr_robot_launch/launch/sr_ur_arm_hand.launch

```diff
@@ -74,7 +76,9 @@
     <!-- HAND -->
     <param name="/hand/mapping/$(arg hand_serial)" value="$(arg hand_id)"/>
     <param name="/hand/joint_prefix/$(arg hand_serial)" value="$(arg hand_id)_"/>
-    <node name="spawn_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model $(arg robot_name)" respawn="false" output="screen"/>
+    <arg if="$(arg start_home)" name="spawn_model_extra_args" value="-unpause -J ra_shoulder_pan_joint 1.147 -J ra_elbow_joint 1.695 -J ra_wrist_1_joint -1.395 -J ra_wrist_2_joint -1.584 -J ra_shoulder_lift_joint -1.926 -J ra_wrist_3_joint 1.830" />    
```

Comment:
I think we should have an arg "home_position" with these values as default but clearly defined so we can change those if we want from the ur10, etc launch file

Reply:
Agreed - I'll make the changes

Reply:
Changes made!